src/Components/Game/Game.js

```diff
@@ -1,111 +1,109 @@
-import React, { useEffect, useState } from 'react';
-import './Game.css'
-import {getSinglePokemon} from '../../ApiCalls'
-import pokeSound from '../../assets/whos-that-pokemon_.mp3'
-import wrongSound from '../../assets/EndCall3.mp3'
-import rightSound from '../../assets/HollowBellNotification.mp3'
+import React, { useEffect, useState, useContext } from 'react';
+import './Game.css';
+import pokeSound from '../../assets/whos-that-pokemon_.mp3';
+import wrongSound from '../../assets/EndCall3.mp3';
+import { AppContext } from '../../AppContext';
+import rightSound from '../../assets/HollowBellNotification.mp3';
 
 
-const Game = ({ get4RdmPokemon}) => {
-  const [singlePokemon, setSinglePokemon] = useState({});
+const Game = ({ get4RdmPokemon, fetchSinglePokemon}) => {
   const [winCounter, setWinCounter] = useState(0);
   const pokemons = get4RdmPokemon();
   const [pokemonChoices, setPokemonChoices] = useState(pokemons);
-  const [bestCounter, setBestCounter] = useState(0)
-  const wrongSoundObj = new Audio(wrongSound)
-  const rightSoundObj = new Audio(rightSound)
-  const [imageClassName, setImageClassName] = useState('single-pokemon')
+  const [bestCounter, setBestCounter] = useState(0);
+  const wrongSoundObj = new Audio(wrongSound);
+  const rightSoundObj = new Audio(rightSound);
+  const [imageClassName, setImageClassName] = useState('single-pokemon');
+  const [isWinner, setIsWinner] = useState(false);
+  const { singlePokemon, setSinglePokemon } = useContext(AppContext);
```

Comment:
Why use the Context API rather than passing `singlePokemon` / `fetchSinglePokemon` down from the App and calling `fetchSinglePokemon` on a useEffect? You could still share the current pokemon across the Pokedex and Game components without needing to bring in Context. Unless there's another place this state is being used?